### ISSUE TITLE
Perform Accessibility Audit

### ISSUE THREAD

**Issue by sabdul14** [FordLabs/PeopleMover#362]
# Feature Request

**As a** FLIP Team member
**I would** like to know how the PeopleMover site is doing in regards to meeting WCAG 2.0 guidelines
**so that** I know what parts of the app I need to adjust to meet accessibility standards & everyone can use our app easily

## Acceptance Criteria

Set of tools identified to perform the accessibility audit
Results from accessibility audit (what needs to be addressed)

## Notes 

Web Content Accessibility Guidelines:
https://www.w3.org/TR/WCAG21/

Ford Accessibility Intro & Testing Deck
https://azureford-my.sharepoint.com/:p:/r/personal/mrygiel_ford_com/_layouts/15/doc2.aspx?sourcedoc=%7BAA18902E-94E1-4BFB-A4DB-CAE1D38D4619%7D&file=Accessibility%20Intro%20and%20Testing%20-%202020-02-14.pptx&action=edit&mobileredirect=true&cid=e4903ce1-e084-4de6-9a44-7ce1185d37cb

(Ask Mike Rygiel for access if needed)

**Comment by michaelrygiel** [comment]
General:
1. Account dropdown shows ? button on top banner in screen reader, needs aria label
2. Cannot use keyboard to sign out from account dropdown menu
3. Hotjar exists in links screen reader menu, try to remove it
4. PeopleMover logo says 'not available' in links screen reader menu, remove that
5. From axe, Element's background color could not be determined due to a background gradient for <h1 class="page-title">PEOPLEMOVER</h1>
6. From axe, ensure all page content is contained by landmarks (main, header, footer, etc)
7. From axe, document must have one main landmark


Space Dashboard:
1. Space buttons do not exists in screen readers currently, need role set on them (role should be button) and aria label
2. Ellipsis on space shows ? button for screen reader, need to add aria label
3. Cannot enter 'edit space modal' using keyboard


Space:
1. From axe, the branding message at the bottom of the page must have sufficient color contrast,
<p class="branding-message">Powered by</p> and <p class="branding-message">FordLabs</p>
2. From axe, Document has multiple elements referenced with ARIA with the same id attribute: sortby-dropdown
3. In edit product modal, hidden element must not contain focusable element, <input type="text" aria-hidden="true" class="hiddenInputField">
4. Cannot see on screen reader button menu or tab to new product button
5. Cannot see on screen reader button menu or tab to edit product button 
6. Cannot see on screen reader button menu or tab to add person button on product
7. Cannot see on screen reader button menu or tab to edit person button on product
8. Cannot see on the calendar date button on screen reader button menu or tab to open calendar
9. React Date Picker has ? button for switching months, non viewable buttons for selecting dates. Although, it is keyboard accessible
10. Revert button in reassignment drawer does not describe what is being reverted, just says 'Revert Button'. Add aria label for what assignment is being reverted
11. Add keyboard accessibility to all of space, which includes highlighting of buttons and inclusion of all buttons/links on page
